src/cache-fs.ts

```diff
@@ -147,47 +169,57 @@ export class CacheFileSystem implements FileSystem {
         });
     }
 
+    dispose() {
```

Comment:
That's what I didn't want to have in the cache implementation but we can put it in for now and regret it later :)